### PR TITLE
Update Language Settings File

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update Language Settings File

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -1,10 +1,9 @@
 $Language = "python"
-$Lang = "python"
 $PackageRepository = "PyPI"
 $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 
-function Extract-python-PkgProperties ($pkgPath, $serviceName, $pkgName)
+function Get-python-PackageInfoFromRepo  ($pkgPath, $serviceDirectory, $pkgName)
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:6]:
```suggestion
function Get-python-PackageInfoFromRepo($pkgPath, $serviceDirectory, $pkgName)
```

File: eng/scripts/Language-Settings.ps1 [hunk 2]
```diff
@@ -47,7 +46,7 @@ function IsPythonPackageVersionPublished($pkgId, $pkgVersion) {
 }
 
 # Parse out package publishing information given a python sdist of ZIP format.
-function Parse-python-Package($pkg, $workingDirectory) {
+function Get-python-PackageInfoFromPackageFile ($pkg, $workingDirectory) {
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:49]:
```suggestion
function Get-python-PackageInfoFromPackageFile($pkg, $workingDirectory) {
```

File: eng/scripts/Language-Settings.ps1 [hunk 3]
```diff
@@ -84,7 +83,7 @@ function Parse-python-Package($pkg, $workingDirectory) {
 }
 
 # Stage and Upload Docs to blob Storage
-function StageAndUpload-python-Docs()
+function Publish-python-GithubIODocs ()
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:86]:
```suggestion
function Publish-python-GithubIODocs()
```

**Comment by chidozieononiwu** [comment]
Merging these since the Changes are not related tot the failures.

**Pull request merged**